eli5/keras/explain_prediction.py

```diff
@@ -20,61 +31,44 @@
 
 # note that keras.models.Sequential subclasses keras.models.Model
 @explain_prediction.register(Model)
-def explain_prediction_keras(estimator, # type: Model
+def explain_prediction_keras(model, # type: Model
                              doc, # type: np.ndarray
-                             target_names=None,
                              targets=None, # type: Optional[list]
                              layer=None, # type: Optional[Union[int, str, Layer]]
-                            ):
+                             image=None,
+                             ):
     # type: (...) -> Explanation
     """
-    Explain the prediction of a Keras image classifier.
+    Explain the prediction of a Keras classifier with the Grad-CAM technique.
 
-    We make two explicit assumptions
-        * The input is images.
-        * The model's task is classification, i.e. final output is class scores.
+    # TODO: explain Grad-CAM.
```

Comment:
Maybe we want to either fix this TODO now, or else remove from docstring, as it would be in the docs?

Reply:
I'll have the description only in the docs to avoid repetition.